src/pages/404.js

```diff
@@ -29,6 +29,7 @@ const PageNotFound = ({location}: Props) => (
               Please contact the owner of the site that linked you to the
               original URL and let them know their link is broken.
             </p>
+            <p><a href="/">Back to home</a></p>
```

Comment:
but always they can click React logo on header and go back to home

Reply:
Yea user can always go home by clicking on the React icon.

Reply:
Yes they can!

Reply:
but for some reasons time to time, it's not obviously

Reply:
> 

Yes you are right @dbarabashdev ! So it good to have a back to home link below for user. What you think about it @harish-sethuraman ?

Reply:
The navigation is mounted on all routes right?, If so there is going to be a way back home on all pages?

Reply:
Ya, but people will doesn't think about going back like that fastly but if there is a back to the home they think going back 
home and exploring the docs again! 

I think back to home link is good for this documentation. I think you understand what I am trying to explain to you.

Reply:
I understand what you are trying to say. 

PS: Change anchor tag to Gatsby Link component.

Reply:
> I understand what you are trying to say.
> 
> PS: Change anchor tag to Gatsby Link component.

Yes i added a Gatsby link component insted of the old anchor tag. But i am not sure that Gatsby link is imported if the Gatsby link is not imported i can add it to the repo.

I mean about this : 
```bash
import { Link } from "gatsby"
```
I am not sure that this link is imported.

Reply:
@harish-sethuraman could u please say about the bug behind preview page failure. May i can fix the error.

Reply:
Try building locally once. I suspect that you didn't add import for `Link`

Reply:
Ok let me try it!

Reply:
@harish-sethuraman 

Fixed all the bugs and now it is working successfully preview page is working and checked whether back to home is working, it is also working successfully. The bug/error was because Gatsby Link was not imported, it has been fixed and now it works.